worlds/metroidprime/__init__.py

```diff
@@ -45,6 +46,7 @@ class MetroidPrimeWorld(World):
     topology_present = True
     item_name_to_id = {name: data.code for name, data in item_table.items()}
     location_name_to_id = every_location
+    settings: MetroidPrimeSettings
```

Comment:
Once I wired this up that settings class worked as expected and correctly asked me to put in an ISO 🎉